windows/setup.yml

```diff
@@ -80,6 +93,34 @@ controller_templates:
             - 'Yes'
             - 'No'
 
+  - name: "WINDOWS / Patching Report"
```

Comment:
This is duplicating an existing job template. Is there a reason not to use the existing template?

Reply:
Here, the point is that all the vars are hardcoded and is run in check mode(no survey). The demo readme shows the scenario. For instance if a manager or a Level1 helpdesk person needs to look at the patching report, the only access they have is to execute this job template within their account on AAP. The idea is to showcase how automation can help elevate lower tiers of support or create self-help automation reports for managers without needing an admin in the path.